invenio_saml/templates/invenio_saml/login_user.html

```diff
@@ -0,0 +1,51 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2015-2020 CERN.
+  Copyright (C)      2021 TU Wien.
+  Copyright (C)      2021 Graz University of Technology.
+
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+{%- extends config.OAUTHCLIENT_LOGIN_USER_TEMPLATE_PARENT %}
+
+{%- from "invenio_oauthclient/_macros.html" import oauth_button %}
+
+
+{%- block form_outer %}
+  {% if config.OAUTHCLIENT_REMOTE_APPS %}
+  <div class="ui form">
+    {% for name in config.OAUTHCLIENT_REMOTE_APPS.keys() %}
+      {{ oauth_button(name, next=request.args.get('next')) }}
+    {% endfor %}
+  </div>
+
+    {%- if config.ACCOUNTS_LOCAL_LOGIN_ENABLED %}
+      <div class="ui horizontal divider">
+        Or
+      </div>
+    {%- endif %}
+  {% endif %}
+
+  {% if config.SSO_SAML_IDPS %}
```

Comment:
I am wondering if we put `Oauth` and `SAML` in the same "block"

- Login with SAML
- Login with Keycloak

OR

user/psw